utility/utils.py

```diff
@@ -1842,6 +1842,28 @@ def configure_kafka_security(rgw_node, cloud_type):
     rgw_node.exec_command(sudo=True, cmd=f"ceph orch restart {rgw_name}")
 
 
+def config_keystone_ldap(rgw_node, cloud_type):
+    """Set the keystone config option on the cluster at startup"""
+    if cloud_type == "openstack":
+        keystone_server = "http://10.0.209.121:5000"
```

Comment:
Avoid hardcoded config server values, read these values from cephci.yaml. This will avoid check for cloud type.